src/Misc/layoutbin/runsvc.sh

```diff
@@ -13,7 +13,7 @@ fi
 # insert anything to setup env when running as a service
 
 # run the host process which keep the listener alive
-./externals/node12/bin/node ./bin/RunnerService.js &
```

Comment:
Do we need some environment to make everything fall back to node12 in case some customer runs into an issue with node 16?

Reply:
Good point. Should it be a comprehensive flag that would force all new references of node16 in this PR back to node12?

Reply:
We should do this for all node16 changes in this pr (including hash files).

`RUNNER_FORCENODEVERSION` that we check the value of and try to use. We should add docs for this as well, and indicate you may be using an unsecure node version (node 12 has an end of life in April)

Reply:
Sounds good, I'll create a constant with that name and comment the meaning there. We could also doc this during our next release. Where else should we document it?